doc-changelog/parse_pr_title.py

```diff
@@ -101,8 +101,10 @@ def changelog_category_cc(cc_type: str):
         "ci": "maintenance",
     }
 
-    # Get the changelog section based on the conventional commit type
-    changelog_section = cc_type_changelog_dict[cc_type]
+    for key, value in cc_type_changelog_dict.items():
+        if key in cc_type:
+            # Get the changelog section based on the conventional commit type
+            changelog_section = cc_type_changelog_dict[key]
```

Comment:
```suggestion
    for key, value in cc_type_changelog_dict.items():
        if key in cc_type:
            # Get the changelog section based on the conventional commit type
            changelog_section = value
            break
```

Reply:
Just leveraging `value` and breaking when a key matches.

Reply:
Ah true, we already have the value. Good catch

Reply:
Completely overseen on my side 😄